src/domain/customer/services/create_customer_service.test.ts

```diff
@@ -0,0 +1,18 @@
+import 'reflect-metadata'
+import Customer from "../model/customer"
+import CustomerRepository from "../repository/customer_repository"
+import CustomerService from "./create_customer_service"
+
+jest.mock('src/domain/customer/model/customer')
+const customerMock = Customer as jest.Mock<Customer>
+const customerRepositoryMock = CustomerRepository as jest.Mock<CustomerRepository>
+describe('CustomerService', () => {
+   it('should call CustomerRepository.create() when CustomerService.create() is called ', () => {
```

Comment:
Você deveria agrupar os testes (`it(...)`) em um segundo describe que é o nome do método. A estrutura é algo como:
```
describe(NomeDaClasse, () => {
    describe(método, () => {
        it(teste, () => {})
    })
})
```

Reply:
refatorado